crates/bevy_pbr/src/render/light.rs

```diff
@@ -875,6 +891,16 @@ pub fn prepare_lights(
             // and ignore rotation because we want the shadow map projections to align with the axes
             let view_translation = GlobalTransform::from_translation(light.transform.translation());
 
+            let cube_face_projection = Mat4::perspective_infinite_reverse_rh(
+                std::f32::consts::FRAC_PI_2,
+                1.0,
+                light.shadow_map_near_z,
+            );
+            let cube_face_rotations = CUBE_MAP_FACES
+                .iter()
+                .map(|CubeMapFace { target, up }| Transform::IDENTITY.looking_at(*target, *up))
+                .collect::<Vec<_>>();
```

Comment:
No dependence on anything in this inner loop so could remain where it was?

Reply:
Reverted the change.